conda_smithy/lint_recipe.py

```diff
@@ -11,9 +11,11 @@
 import jinja2
 import ruamel.yaml
 
-from conda_build.metadata import ensure_valid_license_family
+from conda_build.metadata import ensure_valid_license_family, FIELDS
 
 
+FIELDS['extra'] = ['recipe-maintainers']
```

Comment:
Should we be modifying `conda_build`'s  `FIELDS` or should we make a copy of it first?

Reply:
Yes, also I tried to be a bit more graceful about adding `extra` and `recipe-maintainers` by first checking if `extra` existed and then appended `recipe-maintainers` to the list. (Although maybe the `conda_build` `FIELDS` values should be sets rather than lists?)